src/utilities.js

```diff
@@ -70,6 +70,7 @@ module.exports.buildCLICommand = function buildCLICommand(inputs) {
       `${(inputs.verbose === 'true') ? "--verbose " : ""}` +
       `${(inputs.debug === 'true') ? "--debug " : ""}` +
       `--repo-dir ${inputs.workspace} ` +
+      `--cicd-platform hawk-CICD ` +
```

Comment:
hawk-CICD the identifier we want to use for identifying github actions? What flag are we passing for other cicd platforms?

Reply:
@untra CircleCI Is the only other one but we should also update the new ADO one as well